v22.2/aws-dms.md

```diff
@@ -21,7 +21,21 @@ Complete the following items before starting this tutorial:
 
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.Creating.html) in AWS.
 - Configure a [source endpoint](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html) in AWS pointing to your source database.
-- Ensure you have a secure, publicly available CockroachDB cluster running v22.1.0 or later.
+- Ensure you have a secure, publicly available CockroachDB cluster running v22.2.0 or later, and have created a [SQL user](security-reference/authorization.html#sql-users) that you can use for your AWS DMS [target endpoint](#step-1-create-a-target-endpoint-pointing-to-cockroachdb).
```

Comment:
Wonder if we should just set the min to .4. Do we already have a prereq of creating a unique migration user?

Reply:
+1 on saying that .4 is the minimum